src/commands/cancel.ts

```diff
@@ -0,0 +1,17 @@
+import Command from '../@types/Command'
+
+const cancelShortcut: Command = {
+  id: 'cancel',
+  label: 'Cancel',
+  description: 'Cancel the current gesture.',
+  gesture: undefined,
+  keyboard: { key: ',', meta: true },
+  multicursor: 'ignore',
+  svg: () => null,
+  exec: dispatch => {
+    alert('cancel')
+  },
+  allowExecuteFromModal: true,
```

Comment:
We can omit `allowExecuteFromModal` since gestures cannot be activated when a modal is open.